<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1149</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>1121</width>
      <height>661</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QFrame{
	background-color:rgb(220, 220, 220);
	border:0px solid red;
	border-radius:30px
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame2">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>0</y>
       <width>841</width>
       <height>661</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color:rgb(220, 220, 220);
	border:0px solid red;
	border-radius:30px
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>331</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(255, 165, 142);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="tomato_time">
       <property name="geometry">
        <rect>
         <x>68</x>
         <y>60</y>
         <width>201</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>60</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>25:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="tomato_start">
       <property name="geometry">
        <rect>
         <x>265</x>
         <y>160</y>
         <width>60</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;
image:url(D:/酪梨資料夾/大學作業/專題/pyqt/demo/img/start.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QPushButton" name="tomato_change">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>60</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;
image:url(D:/酪梨資料夾/大學作業/專題/pyqt/demo/img/setting.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="tomato_skip">
       <property name="geometry">
        <rect>
         <x>265</x>
         <y>20</y>
         <width>60</width>
         <height>40</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border: none;
image:url(D:/酪梨資料夾/大學作業/專題/pyqt/demo/img/skip.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
      <widget class="QFrame" name="frame3_1">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>291</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgb(220, 220, 220);</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QLabel" name="tomato_mode">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>0</y>
          <width>261</width>
          <height>91</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Agency FB</family>
          <pointsize>24</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>WORK</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QFrame" name="frame4">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>0</y>
        <width>501</width>
        <height>321</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(153, 153, 153);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="face_tracking">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>461</width>
         <height>281</height>
        </rect>
       </property>
       <property name="text">
        <string>opencv</string>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>330</y>
        <width>841</width>
        <height>331</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:rgb(149, 206, 217);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>120</y>
         <width>131</width>
         <height>101</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>20</pointsize>
        </font>
       </property>
       <property name="text">
        <string>待辦清單</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QFrame" name="user">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>231</width>
       <height>141</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="usericon">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>216</width>
        <height>64</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="user_icon" native="true">
         <property name="minimumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">image:url(D:/酪梨資料夾/大學作業/專題/pyqt/demo/img/user.png);
border-radius:32px</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="username">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>74</y>
        <width>216</width>
        <height>30</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>20</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>user</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QListWidget" name="fuctionlist">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>240</y>
       <width>238</width>
       <height>252</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QListView {
}
QListView::item{
background-color: transparent;
	color: rgb(67, 67, 67);
	padding:10px;
	padding-left:5px;
	margin-bottom:8px;
}
QListView::item:hover {
border-radius:8px;
	background-color: rgb(240, 240, 240)

}
QListView::item:selected {
	background-color: rgb(239, 239, 239);
	color: rgb(12, 20, 36);
border-radius:8px;
}
</string>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="currentRow">
      <number>-1</number>
     </property>
     <item>
      <property name="text">
       <string>首頁</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>img/home.png</normaloff>img/home.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>伸展操</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>img/exercise.png</normaloff>img/exercise.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>新聞</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>img/news.png</normaloff>img/news.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>設定</string>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>img/setting2.png</normaloff>img/setting2.png</iconset>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1149</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
